src/gui/rss/rss_imp.cpp

```diff
@@ -113,19 +113,29 @@ void RSSImp::displayItemsListMenu(const QPoint&)
     QMenu myItemListMenu(this);
     QList<QListWidgetItem*> selectedItems = listArticles->selectedItems();
     if (selectedItems.size() > 0) {
-        bool has_attachment = false;
+        bool hasTorrent = false;
+        bool hasLink = false;
         foreach (const QListWidgetItem* item, selectedItems) {
-            if (m_feedList->getRSSItemFromUrl(item->data(Article::FeedUrlRole).toString())
-                ->getItem(item->data(Article::IdRole).toString())->hasAttachment()) {
-                has_attachment = true;
+            if (!item) continue;
```

Comment:
I'm not sure but I saw it here https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/rss/rss_imp.cpp#L341
I think it's not harmful.